script.js

```diff
@@ -1,28 +1,32 @@
 $(document).ready(function(){
 
-
-
-
   var lightSeq = [];
   var turn = 0;
-  var attempts = 0;
-
+  var attempt = 1;
+  var strictMode = false;
+  var colorIndex = {Red:1,Green:2,Yellow:3,Blue:4};
+//audio
+  var audioCtx = new (window.AudioContext || window.webkitAudioContext)();
+  var oscillator = audioCtx.createOscillator();
+  var gainNode = audioCtx.createGain();
+  oscillator.start();
+  oscillator.type = 'sine';
+  gainNode.connect(audioCtx.destination);
 
 
   $("#Start").click(function(){
-    lightSeq = [];
-    turn = 2;
-    for(var i=0; i<20; i++){
-      lightSeq.push(Math.floor(Math.random()*4));
-    }
+    generateSeq();
```

Comment:
I recommend avoiding the use of abbreviations in code in order to facilitate understanding and maintenance of that code later (by both yourself and other developers that may be working on the code).